### PR TITLE
Add initialization of pairwise Coulomb collisions

### PR DESCRIPTION
**Pull request by roelof-groenewald**
Should wait for the upstream PR #2864 to be merged before merging this.

### CODE REVIEW

File: mewarpx/mewarpx/coulomb_scattering.py [hunk 1]
```diff
@@ -217,3 +217,53 @@ def run_scattering_method(self):
...
+        self.species2 = species2
+        if self.species2 is None:
+            self.species2 = self.species1
+
+        self.log_lambda = log_lambda
+        self.subcycling_steps = subcycling_steps
+
+        print_str = (
+            "Initialized pairwise Coulomb scattering for species "
+            f"{self.species1.name} and {self.species2.name}."
```
Comment (peterscherpelz) [inline mewarpx/mewarpx/coulomb_scattering.py]:
```suggestion
            f"{self.species1.name} and {self.species2.name}.\n"
```

Comment (peterscherpelz) [reply mewarpx/mewarpx/coulomb_scattering.py]: Or a space, whichever you prefer.

**Review by peterscherpelz — APPROVED** [review]
Thanks! I just looked through the mewarpx files assuming the rest will be reviewed upstream.

File: mewarpx/mewarpx/coulomb_scattering.py (hunk 1 above)
Comment (roelof-groenewald) [reply mewarpx/mewarpx/coulomb_scattering.py]: Thanks, I'll change it on the `LangevinElectronIonScattering` print out as well.

**Pull request merged**